src/services/objection.session.service.ts

```diff
@@ -42,6 +43,18 @@ export const retrieveAccessTokenFromSession = (session: Session): string => {
   return token;
 };
 
+export const retrieveObjectionCreateFromObjectionSession = (session: Session): ObjectionCreate => {
+  const objectionsExtraData: ObjectionSessionExtraData = retrieveObjectionSessionFromSession(session);
+  if (objectionsExtraData) {
+    const objectionCreate: ObjectionCreate | undefined = objectionsExtraData.objection_create;
+    if (objectionCreate) {
+      return objectionCreate;
+    }
+    throw new Error("Error retrieving user details for creating objection from objection session");
```

Comment:
This error could maybe say "Error: No ObjectionCreate present in ObjectionSession"

Reply:
Changed message.